packages/arb-bridge-eth/contracts/rollup/INode.sol

```diff
@@ -60,7 +60,7 @@ interface INode {
 
     function latestChildNumber() external view returns (uint256);
 
-    function requirePastDeadline() external view;
+    function requirePastDeadline(uint256 blocksSpentPaused) external view;
```

Comment:
We might want to consider having the node query from the rollup to make the API simpler